french/maps/outback_01.rpy

```diff
@@ -1,146 +1,145 @@
 
 # game/maps/outback_01.rpy:208
 translate french outback_interact_01_e45b9947:
 
     # d nulla "Also, {i}definitely{/i} just for low level Grimm.{w=0.3} I mean..."
-    d nulla ""
+    d nulla "Aussi, {i}définitivement{/i} juste pour du Grimm de bas niveau.{w=0.3} Je veux dire..."
 
 # game/maps/outback_01.rpy:212
 translate french outback_interact_01_0d668c10:
 
     # d nulla "The circle of pebbles around the blast radius is one thing, but 'MINE' painted in bright red with a giant arrow on the nearby rock?"
-    d nulla ""
+    d nulla "Le cercle de cailloux autour du rayon d'explosion est une chose, mais 'A MWA' peint en rouge brillant avec un arc géant sur le rocher à côté ?"
 
 # game/maps/outback_01.rpy:213
 translate french outback_interact_01_c786a6bf:
 
     # p nulla "She likes to be thorough, I suppose...?"
-    p nulla ""
+    p nulla "Elle aime être minutieuse, je suppose... ?"
 
 # game/maps/outback_01.rpy:215
 translate french outback_interact_01_2bb621e6:
 
     # d nulla "Thankful that I don't need to watch my every step but doesn't this kinda defeat the purpose?"
-    d nulla ""
+    d nulla "Je suis reconnaissant de ne pas avoir besoin de surveiller chacun de mes pas, mais est-ce que ça ne va pas à l'encontre du but recherché ?"
 
 # game/maps/outback_01.rpy:217
 translate french outback_interact_01_92e2f7c6:
 
     # g_st c_mili neutral glitch "Observation.{w=0.3} She may use the clearly marked one to lull enemies into a false sense of security for any hidden ones."
-    g_st c_mili neutral glitch ""
+    g_st c_mili neutral glitch "Observation.{w=0.3} Elle peut utiliser celui clairement indiqué pour induire les ennemis dans un faux sentiment de sécurité pour ceux qui sont cachés."
 
 # game/maps/outback_01.rpy:247
 translate french outback_talk_02_979ea004:
 
     # j surprise "Hey, carrots.{w=0.3} Can I ask you something?"
-    j surprise ""
+    j surprise "Hey, carrotes.{w=0.3} Est-ce que je peux te demander quelque chose ?"
 
 # game/maps/outback_01.rpy:248
 translate french outback_talk_02_01d1643f:
 
     # e surprise "Hmmm?"
-    e surprise ""
+    e surprise "Hmmm ?"
 
 # game/maps/outback_01.rpy:255
 translate french outback_talk_02_296a786c:
 
     # j nulla "I was thinking about something you said earlier...{w=0.3} How you could tell the mines are gunpowder based because of the smell.{w=0.3} How do you know what gunpowder smells like?"
-    j nulla ""
+    j nulla "Je pensais à quelque chose que tu as dis plus tôt...{w=0.3} Comment peux-tu dire que les mines fonctionnent à partir de poudre à canon à partir de l'odeur.{w=0.3} Comment sais-tu ce que sent la poudre à canon ?"
 
 # game/maps/outback_01.rpy:257
 translate french outback_talk_02_e30210d6:
 
     # e nulla "Oh, that?{w=0.3} Easy, my uncle used to make fireworks, back when I was young."
-    e nulla ""
+    e nulla "Oh, ça ?{w=0.3} Facile, mon oncle a déjà fait des feux d'artifices, quand j'étais petite."
```

Comment:
used to -> avait l'habitude de

mon oncle avait l'habitude de faire des feux d'artifices